Userland/Libraries/LibCore/InputBitStream.h

```diff
@@ -99,9 +99,9 @@ class BigEndianInputBitStream : public Stream {
                         m_current_byte.clear();
                 }
             } else {
-                auto temp_buffer = TRY(ByteBuffer::create_uninitialized(1));
-                TRY(m_stream.read(temp_buffer.bytes()));
-                m_current_byte = temp_buffer[0];
+                u8 temp_stream_value;
+                TRY(m_stream.read({ &temp_stream_value, sizeof(temp_stream_value) }));
+                m_current_byte = temp_stream_value;
```

Comment:
How about ```TRY(m_stream.read({ &m_current_byte, sizeof(m_current_byte) }));```?

Reply:
Can't do that, because `m_current_byte` isn't a `u8`, but an `Optional<u8>`. Can't use the `operator*` and take the address of it either, as if it doesn't have a value then it will `VERIFY(m_has_value)`.